src/test/java/com/example/datavisualizationbackend/ApplicationTests.java

```diff
@@ -17,21 +17,21 @@
 @RunWith(SpringJUnit4ClassRunner.class)
 public class ApplicationTests {
```

Comment:
The standard convention is to have a test class for each class you are testing.

So this class should be UploadEventControllerTest and moved to the same package as the class it is testing (but still in the test root directory).